__tests__/cmds/versions.test.js

```diff
@@ -137,11 +135,32 @@ describe('rdme versions*', () => {
 
       const mockRequest = getApiNock().post('/api/v1/version').basicAuth({ user: key }).reply(400, errorResponse);
 
-      await expect(createVersion.run({ key, version, fork: '0.0.5' })).rejects.toStrictEqual(
-        new APIError(errorResponse)
-      );
+      await expect(createVersion.run(args)).rejects.toThrow(new APIError(errorResponse));
       mockRequest.done();
     });
+
+    describe('ci environments', () => {
+      beforeEach(() => {
+        process.env.CIRCLECI = true;
```

Comment:
Purposefully testing it with `CIRCLECI` here instead of `GITHUB_ACTIONS` because I know that `GITHUB_ACTIONS` will always be present in our CI and I don't want to delete that from the workflow runner.

Reply:
good thinking!